spec/requests/notifications_request_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "Notifications", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get '/notifications'
+      expect(response).to redirect_to(user_session_path)
+    end
+  end
+end
```

Comment:
I think this needs to be fixed. We are authenticating_user for Notification controller actions but I'm not sure how to simulate logging in within a request spec (devise login_user helpers don't work in request specs). I will dig into this

Reply:
I wonder if you can create a user with FactoryBot and then use login_as(user)

```
let(:user) { FactoryBot.create(:user) }

it "can get to notifications when logged in" do
login_as(user)
get .....
end
it "gets redirected to login when not logged in" do
get....
end
```

Reply:
Oh, someone already did this in the latest. Cool!